main/dapps/dapp-templates.md

```diff
@@ -0,0 +1,56 @@
+## Dapp Templates
+
+When creating a new dapp using `agoric init`, you have the option of
```

Comment:
We really should decide how we're capitalizing d a p p; I think we've got all of "dapp", "Dapp", and "dApp" in various places. ( "Dapp" was used above at line 43 in dapps/README.md)

Reply:
I think it makes sense to use `dapp` in most usage, and `Dapp` in a title or proper noun, like Fungible Faucet Dapp. I don't think we should use dApp.